Added steps for running debug script, cleaned up topic. (#214)
mulesoft/docs-runtime-manager: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc

endif::[]
:keywords: agent, mule, servers, monitor, notifications, external systems, third party, get status, metrics

> You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot Runtime Manager agent installation issues. If you experience issues with the Runtime Manager agent plugin or with managing applications on Anypoint Platform, run the script and save the output before filing a support ticket.

Reviewer: ```suggestion
You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot Anypoint Runtime Manager agent installation issues. If you experience issues with the Runtime Manager agent plug-in or with managing applications on Anypoint Platform, run the script and save the output before filing a support ticket.
```

> The `amc_debug` script performs the following:

Reviewer: ```suggestion
The `amc_debug` script performs the following actions:
```

. Discovers the Mule agent JVM process
. Extracts the thread dump (not forced if possible)
. Optionally extracts the heap dump (if you specify the `-heap` option)
+
This step can cause delays in running applications.
+
. Extracts the Java version, CPU usage, memory usage, and disk usage
. Copies all logs
. Copies Runtime Manager agent configuration files
. Archives all the information collected in a .zip file

== Run the amc_debug Script

To run the `amc_debug` script to collect information about the agent installation:

. Open a terminal window and change to the `$MULE_HOME/bin` folder.
. Run the script:
+
`./amc_debug`
+
Include the `-heap` option to extract the heap in addition to the other data.
+

> After the script completes, the folder `$MULE_HOME/bin` contains a .zip file with a name similar to `snapshot_201910081712.zip`.

Reviewer: ```suggestion
After the script finishes, the folder `$MULE_HOME/bin` contains a .zip file with a name similar to `snapshot_201910081712.zip`.
```
"completes" requires an object; "is complete" is OK. See style ref.

> . Send the .zip file to MuleSoft support engineers.

Reviewer: Earlier referred to as just "Support." Standardize?
